JavaDirectory/src/main/java/com/bug/free/invention/api/controllers/US002.java

```diff
@@ -0,0 +1,92 @@
+            returnList.add(new Employee(1,"name","difjidsfj"));
+            returnList.add(new Employee(2,"ijisdfjdsif","dsuifjhduisfhui"));
+            return returnList;
+        };
+
+/*    @GetMapping("/jobCapability")
+    public String  getJobCapability(){
+        System.out.println("Attempting connection");
+        try{
+            Statement statement = DBConfig.getConnection().createStatement();
+            String dbQuery = "SELECT Job.Job_Title, Capability.capabilityName FROM Job JOIN Capability ON(Job.capabilityID = Capability.capabilityID);";
+            ResultSet results = statement.executeQuery(dbQuery);
+            JSONObject return_json= new JSONObject();
+            int i = 0;
+            while(results.next()) {
+                String job_title = results.getString("Job_Title");
+                String capabilityname = results.getString("capabilityName");
+                JSONObject tempJson = new JSONObject("{\"job_title\": \"" + job_title + "\", \"capabilityname\": \""+ capabilityname+ "\"}");
+                return_json.put(String.valueOf(i), tempJson);
+                i++;
+            }
+            return return_json.toString();
+        }
+        catch (Exception e){
+            e.printStackTrace();
+            return null;
+        }
+    }*/
+
+        @GetMapping("/jobRoles") //US001
+        public List<Job> getJobRoles(){
+            System.out.println("Attempting connection");
+            try{
+
+                Statement statement = DBConfig.getConnection().createStatement();
+                String dbQuery = "SELECT job.jobID, job.jobTitle, job.bandID, capability.capabilityID, capability.capabilityName, band.bandName, band.bandLevel, jobFamily.jobFamilyTitle FROM job JOIN capability ON(job.capabilityID = capability.capabilityID) JOIN band ON(job.bandID = band.bandID) JOIN jobFamily ON(job.jobFamilyID = jobFamily.jobFamilyID);";
+                ResultSet results = statement.executeQuery(dbQuery);
+                List<Job> Jobs = new ArrayList<Job>();
+                
+                while(results.next()){
+                    Jobs.add(new Job(results.getInt("jobID"),results.getString("jobTitle"),
+                            results.getInt("capabilityID"),results.getInt("bandID"), results.getString("capabilityName"),
+                            results.getString("bandName"), results.getInt("bandLevel"), results.getString("jobFamilyTitle")));
+
+                }
+                return Jobs;
+            }
+            catch (Exception e){
+                e.printStackTrace();
+                return null;
+            }
+        };
+        @GetMapping("/jobSpec")
+        public String getJobSpecLink(@RequestParam Integer JobID){
+            String dbQuery = "SELECT 'jobSpec' FROM `Job` WHERE 'jobID' = ?;";
+            try(Connection DatabaseConnection = DBConfig.getConnection()){
+                PreparedStatement FindJobSpecById = DatabaseConnection.prepareStatement(dbQuery);
```

Comment:
Variables should not be Capitalised
```suggestion
                PreparedStatement findJobSpecById = DatabaseConnection.prepareStatement(dbQuery);
```